test/e2e/garbage_collector.go

```diff
@@ -72,6 +106,30 @@ func newOwnerRC(f *framework.Framework, name string) *v1.ReplicationController {
 	}
 }
 
+// verifyRemainingObjects verifies if the number of the remaining deployment
```

Comment:
nit: keep the name consistent. Also consider renaming to verifyRemainingDeploymentsAndReplicasets?